config/experimental/s3configsource/factory.go

```diff
@@ -0,0 +1,55 @@
+package s3configsource
+
+import (
+	"context"
+	"errors"
+
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/experimental/configsource"
+
+	splunkprovider "github.com/signalfx/splunk-otel-collector"
```

Comment:
Ditto, why are we importing github.com/signalfx/splunk-otel-collector here?

Reply:
I've replaced it locally with the configprovider folder, it's being imported because there's still a dependency on a single file that hasn't been merged into core yet